generics/helpers/utils.js

```diff
@@ -298,9 +298,22 @@ function convertArrayObjectIdtoStringOfObjectId(ids) {
 }
 
 function getTimeWith530HrDifference(time){
-  time.setHours(time.getHours()-5);
-  time.setMinutes(time.getMinutes()-30);
-  return time
+  let localTimeZone = process.env.TIMEZONE_DIFFRENECE_BETWEEN_LOCAL_TIME_AND_UTC
+  let localTime = localTimeZone.split(":")
+  let localHourDifference = Number(localTime[0])
+  let getTimeDiffInMinutes = localHourDifference * 60 + ((localHourDifference/ Math.abs(localHourDifference))*Number(localTime[1]))
```

Comment:
@ankitpws Add indentation and code comments wherever possible

Reply:
done